package/gluon-wlan-encryption-psk/Readme.md

```diff
@@ -0,0 +1,15 @@
+# Gluon wlan encrypt
```

Comment:
I think this should be remove here and add to the docs

Reply:
Yes, it better fits in the docs and "Gluon WLAN encryption"